test/README.md

```diff
@@ -85,14 +81,15 @@ the `-run` flag with `go test`
 go test -v ./test/... -tags=e2e -run ^TestPipelinesE2E
 ```
 
-### Prerequisite to run make check and make generated 
+### Prerequisite to run make check and make generated
 
-Ensure that golangci-lint and yamllint are installed on your development machine. If not then install them following [golangci-lint](https://golangci-lint.run/usage/install/) and [yamllint](https://yamllint.readthedocs.io/en/stable/quickstart.html).
+Ensure that golangci-lint and yamllint are installed on your development machine. If not then install them following [golangci-lint](https://golangci-lint.run/welcome/install/) and yamllint.
```

Comment:
Any reason we're removing the link to `yamllint`?

Reply:
@aThorp96 Thanks a lot for taking a look at it, and I update it.